src/modules/Builder/components/Configuration/components/Network/style.scss

```diff
@@ -0,0 +1,3 @@
+.custom-control {
```

Comment:
You need to make that these styles are applied only in the `Network` component. Otherwise, they will be applied in other components.